src/node.cc

```diff
@@ -4188,6 +4188,10 @@ void Init(int* argc,
   V8::SetFlagsFromString(NODE_V8_OPTIONS, sizeof(NODE_V8_OPTIONS) - 1);
 #endif
 
+  // Allow for environment set preserving symlinks
+  if (secure_getenv("NODE_PRESERVE_SYMLINKS")) {
+     config_preserve_symlinks = true;
+  }
```

Comment:
Syntax fixed.

What you suggest is the best way to parse the flag, any value except for empty string? Should it handle ```env NODE_PRESERVE_SYMLINKS=false``` ?